### ISSUE TITLE
SysML InterfaceBlock support

### ISSUE THREAD

**Issue by sz332** [gaphor/gaphor#2570]
Is your feature request related to a problem?

Currently it is not possible to create an InterfaceBlock on the diagram using the SysML profile.

Describe the solution you'd like

I would like to be able to create an InterfaceBlock using the toolbar.

**Comment by marek-piirikivi** [comment]
Metamodel defines the element, but it lacks diagram representation and toolbox option.

**Comment by tompkins-ct** [comment]
This is the same with a number of items including:
- view
- viewpoint
- stakeholder
- testcase 

Since these are all in the metamodel, it might be simpler for someone to add them to the toolbox since their representation is more or less derived from the blocks.

**Comment by sz332** [comment]
@tompkins-ct can you please create a separate ticket for each missing item that you would like to have? (the reason for separate tickets is so that we can work in parallel on the topic). Thanks!

**Comment by sz332** [comment]
Started working on this task.

**Comment by sz332** [comment]
Merged to main, closing task.
